index.d.ts

```diff
@@ -0,0 +1,55 @@
+type RecordObject = import('cql-execution').RecordObject;
+type PatientObject = import('cql-execution').PatientObject;
+type DataProvider = import('cql-execution').DataProvider;
+type RetrieveDetails = import('cql-execution').RetrieveDetails;
+type Code = import('cql-execution').Code;
+type Date = import('cql-execution').Date;
+type Interval = import('cql-execution').Interval;
+
+class FHIRObject implements RecordObject {
+  constructor(json: any, typeInfo: any, modelInfo: any);
+  get(field: string): any;
+  getId(): string;
+  getCode(field: string): Code;
+  getDate(field: string): Date;
+  getInterval(field: string): Interval;
+  getDateOrInterval(field: string): Date | Interval;
+  _is(typeSpecifier: any): boolean;
+  _typeHierarchy(): any;
+  getTypeInfo(): any;
+}
+
+class Patient extends FHIRObject implements PatientObject {
+  constructor(bundle: any, modelInfo: any, patientSourceOptions: PatientSourceOptions);
+  findRecords(profile: string, retrieveDetails?: RetrieveDetails): FHIRObject;
+  findRecord(profile: string, retrieveDetails?: RetrieveDetails): FHIRObject;
```

Comment:
So this is odd... I actually don't know why this function even exists. In this project (`cql-exec-fhir`) it's only used in tests.  In `cql-execution`, it is not used at all (and is _not_ part of the `PatientObject` interface). So... do you know why we even have this function at all?

Reply:
Hah, yeah. This has always been in cql-exec-fhir since I started working on it. I wonder if there were initially intentions to support a `findRecord` call from cql-execution in cases where you know there should only be one record or you only want one record?

But yeah, I'm not sure why the function still exists. I personally have never used it for anything